### PR TITLE
libtmux: 0.10.3 -> 0.11.0, tmuxp: 1.9.2 -> 1.11.0

### PR DESCRIPTION
**Pull request by otavio**
- libtmux: 0.10.3 -> 0.11.0
- tmuxp: 1.9.2 -> 1.11.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/libtmux/default.nix [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
 
   src = fetchFromGitHub {
     owner = "tmux-python";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256:0syj8m4x2mcq96b76b7h75dsmcai22m15pfgkk90rpg7rp6sn772";
+    hash = "sha256-QbKqS40la6UGZENyGEw5kXigzexp3q7ff43fKlQ9GqE=";
   };
 
   nativeBuildInputs = [
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/libtmux/default.nix:21]: Please dont use pkgs as an input.

File: pkgs/tools/misc/tmuxp/default.nix [hunk 2]
```diff
@@ -6,11 +6,11 @@ let
...
-  version = "1.9.2";
+  version = "1.11.0";
 
   src = pypkgs.fetchPypi {
     inherit pname version;
-    sha256 = "sha256-3RlTbIq7UGvEESMvncq97bhjJw8O4m+0aFVZgBQOwkM=";
+    sha256 = "sha256-N5kZ+e17ZgLOCvV/lcT/hdG1VNqLxh98QOQyM0BmZCA=";
   };
 
   # No tests in archive
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/tmuxp/default.nix:16]: But there are tests on GitHub. Can we run those?

Comment (otavio) [reply pkgs/tools/misc/tmuxp/default.nix:16]: I am not intending to change how the package is done but upgrade it to fix the build error now in master.

**Review by Misterio77 — APPROVED** [review]
LGTM code-wise, binary builds and runs as expected. Running tests would be good, but I think it makes sense to do on a follow up PR (so the bump isn't blocked).

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 170276 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>4 packages built:</summary>  <ul>    <li>arsenal</li>    <li>python310Packages.libtmux</li>    <li>python39Packages.libtmux</li>    <li>tmuxp</li>  </ul></details>

File: pkgs/tools/misc/tmuxp/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/tools/misc/tmuxp/default.nix:16]: I think that's fine for now. Still strictly better

**Pull request merged**